services/redis/redis.go

```diff
@@ -77,24 +79,100 @@ func (s *redisService) SetChannel(c pushers.Channel) {
 	s.ch = c
 }
 
+type redisDatum struct {
+	DataType byte
+	Content  interface{}
+}
+
+func (d *redisDatum) ToString() (value string, success bool) {
+	switch d.DataType {
+	case 0x2b:
+		fallthrough
```

Comment:
@CapacitorSet  Why do you use a fallthrough here? Ty

Reply:
The parser will put a string in Content both if it receives a simple string `0x2b` (line 120, it uses `cmd` which is of type string) or if it receives a bulk string `0x24` (line 129, `str` is of type string). Therefore, in both cases we use the same logic, a type assertion to string.